core/src/apps/solana/sign_message.py

```
@@ -0,0 +1,133 @@
+            bw, msg.application_domain, _APPLICATION_DOMAIN_LENGTH
+        )
+    writers.write_uint8(bw, msg.message_format)
+    if msg.application_domain:
+        writers.write_uint8(bw, _SIGNER_COUNT)
+        writers.write_bytes_fixed(bw, signer_pub_key_bytes, _PUBLIC_KEY_LENGTH)
+    writers.write_uint16_le(bw, message_length)
+    writers.write_bytes_unchecked(bw, msg.message)
+    return bytes(bw.buffer)
+
+
+def sanitize_message(msg: SolanaSignMessage):
```

Ruff (0.8.2)

Missing return type annotation for public function `sanitize_message`

Add return type annotation: `None`

(ANN201)